src/zebras/_period.py

```diff
@@ -64,6 +64,10 @@ class _BasePeriod:
     data: Any
     freq: Freq
 
+    @property
+    def freqstr(self) -> str:
+        return self.freq.name
```

Comment:
```suggestion
        return str(self.freq)
```

`.name` does not include the number part, e.g. `zb.freq("3D").name == "D"`